cmake/modules/AddSwift.cmake

```diff
@@ -1528,7 +1528,10 @@ function(add_swift_library name)
                  "${SWIFT_${sdk}_ICU_UC}")
             # temporary fix for atomic needing to be
             # after object files for libswiftCore.so
-            if("${sdk}" STREQUAL "ANDROID")
+            if("${sdk}" STREQUAL "FREEBSD")
+              list(APPEND swiftlib_private_link_libraries_targets
```

Comment:
Can you please add a comment here explaining why we need this?

Reply:
I thought the existing comment is enough, it's the same issue but we use builtins instead of `-latomic`. I don't know how to explain this or the reason. 😊

Reply:
You should update the comment, then.

Reply:
I mean it imports `__atomic_store` `__atomic_compare_exchange` `__atomic_load` from builtins. should i just list them there?